pyproject.toml

```diff
@@ -97,6 +97,7 @@ force-exclude = '''
 [tool.mypy]
 check_untyped_defs = true
 ignore_missing_imports = true
+show_error_codes = true
```

Comment:
this [shows error codes](https://mypy.readthedocs.io/en/stable/error_codes.html#displaying-error-codes), which was useful for our flake8 lint that requires `type: ignore[error-code]`